tools/release/image-mirror/mirror.go

```diff
@@ -82,10 +71,8 @@ func (m *mirror) work() {
 func (m *mirror) getRemoteTags() {
 	var url string
 	switch m.sourceRepo.Host {
-	case ghcr:
-		url = fmt.Sprintf("https://ghcr.io/v2/%s/tags/list", m.sourceRepositoryName())
-	case gcr:
-		url = fmt.Sprintf("https://gcr.io/v2/%s/tags/list", m.sourceRepositoryName())
```

Comment:
Why is support for ghcr and gcr being removed?

Reply:
We removed the mirroring from the repository at the time of decoupling the operator releases, I think. I am not able to find the use cases for ghcr and gcr in this repo for mirroring tool, so removing the unused code.

Reply:
I don't think we need to remove this at the moment. Is there a use case where we may need to mirror an image from gcr or ghcr in the future?

Reply:
Ok

Reply:
One final comment on this. Just because this isn't actively used doesn't make it "dead code". You are removing functionality of a tool here. If we ever needed to mirror GHCR or GCR images again the next person would have to re-add these bits that you have removed.

I am not willing to block the PR for this but I don't think it's fair to classify this as `dead code` in practice.